docs/partials/api.md

```diff
@@ -163,6 +163,22 @@ Use the `emphasis` attribute to apply border highlights to the `auro-accordion`
 
 </auro-accordion>
 
+### disabled
```

Comment:
This is in the wrong spot in the doc. Currently it's under the examples for the accordion group and should be moved up in alphabetical order with the regular accordion examples.